app-ui-catalog/src/main/java/app/k9mail/ui/catalog/ui/common/PagedContent.kt

```diff
@@ -56,7 +57,9 @@ fun <T> PagedContent(
             HorizontalPager(
                 pageCount = pages.size,
                 state = pagerState,
-                modifier = Modifier.fillMaxSize(),
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(bottom = MainTheme.sizes.topBarHeight),
```

Comment:
This is using the wrong value. Some padding is necessary because currently we're not applying the window insets automatically. If we want to keep doing it manually, it should be at a higher level. But since we're currently not drawing behind the status bar or navigation bar, maybe we'll just use `WindowCompat.setDecorFitsSystemWindows(window, true)` in `CatalogActivity`?